modules/runbot/runbot.js

```diff
@@ -3,42 +3,42 @@ var Sandbox = require.safe("sandbox"),
 
 exports.run = function(api, event) {
     sandbox.run(event.arguments_body, function(output) {
-		var out = '';
+        var out = '';
 
-		if (output.console.length > 0) {
-			out += 'Console Output:\n-----------------------\n';
-		}
+        if (output.console.length > 0) {
+            out += 'Console Output:\n-----------------------\n';
+        }
 
-		for (var i = 0; i < output.console.length; i++) {
-			out += output.console[i] + '\n';
-		}
+        for (var i = 0; i < output.console.length; i++) {
+            out += output.console[i] + '\n';
+        }
 
-		if (output.result !== 'null') {
-			if (output.console.length > 0) {
-				out += '\n';
-			}
-			out += 'Result:\n-----------------------\n';
-			switch (output.result) {
-				case 'TimeoutError': {
-					out += 'Time limit exceeded (500ms).\n';
-					break;
-				}
-				case 'Error':
-				case '\'undefined: undefined\'': {
-					out += 'An exception was thrown.\n';
-					break;
-				}
-				default: {
-					out += output.result + '\n';
-					break;
-				}
-			}
-		}
+        if (output.result !== 'null') {
+            if (output.console.length > 0) {
+                out += '\n';
+            }
+            out += 'Result:\n-----------------------\n';
+            switch (output.result) {
+                case 'TimeoutError': {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 12 space characters but found 16.](https://www.codacy.com/app/theprojectshop/Kassy/file/2555194762/issues/source?bid=3282987&fileBranchId=3291254#l22)